homeassistant/components/landisgyr_heat_meter/sensor.py

```diff
@@ -256,7 +320,8 @@ async def async_setup_entry(
 
 
 class HeatMeterSensor(
-    CoordinatorEntity[DataUpdateCoordinator[HeatMeterResponse]], RestoreSensor
+    CoordinatorEntity[DataUpdateCoordinator[HeatMeterResponse]],
+    SensorEntity,
 ):
     """Representation of a Sensor."""
```

Comment:
```suggestion
    CoordinatorEntity[DataUpdateCoordinator[HeatMeterResponse]],
    SensorEntity,
):
    """Representation of a Sensor."""
    
    entity_description: HeatMeterSensorEntityDescription
```
and also use `__init__(..., description: HeatMeterSensorEntityDescription, ...` so you can get rid of `# type: ignore[attr-defined]` when using `value_fn()`.